http/health_test.go

```diff
@@ -26,28 +27,41 @@ func TestHealthHandler(t *testing.T) {
 			wants: wants{
 				statusCode:  http.StatusOK,
 				contentType: "application/json; charset=utf-8",
-				body:        `{"name":"influxdb", "message":"ready for queries and writes", "status":"pass", "checks":[]}`,
+				status:      "pass",
 			},
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			HealthHandler(tt.w, tt.r)
 			res := tt.w.Result()
-			content := res.Header.Get("Content-Type")
+			contentType := res.Header.Get("Content-Type")
 			body, _ := ioutil.ReadAll(res.Body)
 
 			if res.StatusCode != tt.wants.statusCode {
 				t.Errorf("%q. HealthHandler() = %v, want %v", tt.name, res.StatusCode, tt.wants.statusCode)
 			}
-			if tt.wants.contentType != "" && content != tt.wants.contentType {
-				t.Errorf("%q. HealthHandler() = %v, want %v", tt.name, content, tt.wants.contentType)
+			if tt.wants.contentType != "" && contentType != tt.wants.contentType {
+				t.Errorf("%q. HealthHandler() = %v, want %v", tt.name, contentType, tt.wants.contentType)
 			}
-			if tt.wants.body != "" {
-				if eq, diff, err := jsonEqual(string(body), tt.wants.body); err != nil {
-					t.Errorf("%q, HealthHandler(). error unmarshaling json %v", tt.name, err)
-				} else if !eq {
-					t.Errorf("%q. HealthHandler() = ***%s***", tt.name, diff)
+			var content map[string]interface{}
+			if err := json.Unmarshal(body, &content); err != nil {
+				t.Errorf("%q, HealthHandler(). error unmarshaling json %v", tt.name, err)
```

Comment:
Maybe add a `continue` after this so you can remove the following `else`. More of a nit than request.

Reply:
The upside of `continue` over `return` or `t.Fatalf` is that the other tests would continue. I guess it doesn't matter here since there is only one test in the slice.

Reply:
**return** from t.Run handler is the way to avoid else branching herein ... repaired.